### PR TITLE
.organizationmap: add Xin Yuan and Yichao Li's affiliation

### PR DESCRIPTION
**Pull request by dragonylffly**
Signed-off-by: Li Wang <laurence.liwang@gmail.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
retest this please.

File: .organizationmap [hunk 1]
```diff
@@ -98,6 +98,8 @@ Debian GNU/Linux <contact@debian.org> Thorsten Glaser <tg@mirbsd.de>
 Deutsche Telekom <contact@telekom.de> Daniel Gollub <d.gollub@telekom.de>
 Deutsche Telekom <contact@telekom.de> Danny Al-Gaaf <danny.al-gaaf@bisect.de>
 Deutsche Telekom <contact@telekom.de> Thomas Bechtold <t.bechtold@telekom.de>
+DiDi <contact@didichuxing.com> Xin Yuan <mychoxin@gmail.com>
+DiDi <contact@didichuxing.com> Yichao Li <liyichao.good@gmail.com>
```
Comment (tchaikov) [inline .organizationmap]: @dragonylffly i cannot find "Yichao Li" in the output of "git log", are you sure it's the right name and mail address?

Comment (tchaikov) [reply .organizationmap]: could you update .mailmap also? see http://tracker.ceph.com/projects/ceph/wiki/Ceph_contributors_list_maintenance_guide

Comment (dragonylffly) [reply .organizationmap]: updated

**Comment by dragonylffly** [comment]
@tchaikov

**Pull request merged**